content/general/queues.textile

```diff
@@ -46,6 +46,8 @@ Ably combines both pub/sub and queueing functionality in its "platform":https://
   <img src="/images/diagrams/ably-queues-overview.png" style="width: 100%" alt="Ably Queues overview">
 </a>
 
+**Note:** It's not a requirement for this scenario, but Ably has the "Asset Tracking SDK":/asset-tracking, which can provide a complete and convenient asset tracking solution. You can read more about what's provided in the "docs":/asset-tracking.
```

Comment:
Not sure you need the second sentence here?

Reply:
I've changed it a bit. I decided it was worth guiding to both the product page and the docs. The product page also has a nice big box linking to the docs too.